SQL.script

```diff
@@ -1,4 +1,118 @@
-Create Database SocialMedia
```

Comment:
Oh I tried running that on a new DB in SQL Server Mangement Studio so I deleted it. Forgot to put in back in when copy-paste.
If I were to add those line, a separate query with those line will automatically create a DB right instead of me going through the old fashion way of creating a new DB and insert those SQL Script specific for the DB